### PR TITLE
Improve the driver configs contribution documentation

### PR DESCRIPTION
**Pull request by maxgio92**
This PR tries to improve the guide for contributing DBG configurations and link to it on the home of this repository.

### CODE REVIEW

File: driverkit/README.md [hunk 1]
```diff
@@ -98,26 +98,41 @@ make -e TARGET_DISTRO=debian stats
...
+- From the `driverkit` directory of the repository, run:
+  ```shell
+  make generate -e TARGET_DISTRO=<TARGET_DISTRO> -e TARGET_KERNEL=<TARGET_KERNEL>
+  ```
+  > Besides `$TARGET_DISTRO` and `$TARGET_KERNEL`, you can find more filters [here](#available-filters), that help you to target specific kernel releases.
+  >
+  > If you want the drivers for your local kernel, you can fill `$TARGET_KERNEL` with: `"$(uname -r)_$(uname -v | sed 's/#\([[:digit:]]\+\).*/\1/')"`.
+  >
+  > Available values for `$TARGET_DISTRO` can be found [here](https://github.com/falcosecurity/driverkit#supported-targets).
+
```
Comment (FedeDP) [inline driverkit/README.md:121]: I would also explain how to add "kernelurls" config key to the created conf file to make sure that driverkit is able to find and download them, if possible!

Comment (maxgio92) [reply driverkit/README.md:121]: The scope of this answer is a quick guide.
I'd instead put a detailed reference documentation about driverkit into its repo and linking to it here. WDYT @FedeDP?

Comment (FedeDP) [reply driverkit/README.md:121]: Yep, makes sense, thank you!

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**